content/en/docs/reference/glossary/cluster.md

```diff
@@ -4,20 +4,19 @@ id: cluster
 date: 2019-06-15
 full_link: 
 short_description: >
-   A set of worker machines, called nodes, that run containerized applications. Every cluster has at least one worker node.
-
+   A set of machines hosting containerized workloads, in the form of pods, and the control plane that manages them.  
 aka: 
 tags:
 - fundamental
 - operation
 ---
-A set of worker machines, called {{< glossary_tooltip text="nodes" term_id="node" >}},
-that run containerized applications. Every cluster has at least one worker node.
+A set of machines hosting containerized {{< glossary_tooltip text="workloads" term_id="workload" >}}, in the form of {{< glossary_tooltip text="pods" term_id="pod" >}}, and the {{< glossary_tooltip text="control plane" term_id="control-plane" >}} that manages them.  
 
 <!--more-->
-The worker node(s) host the {{< glossary_tooltip text="Pods" term_id="pod" >}} that are
-the components of the application workload. The
-{{< glossary_tooltip text="control plane" term_id="control-plane" >}} manages the worker
-nodes and the Pods in the cluster. In production environments, the control plane usually
-runs across multiple computers and a cluster usually runs multiple nodes, providing
-fault-tolerance and high availability.
+Machines may be physical or virtual.  Machines that host pods are referred to as {{< glossary_tooltip text="nodes" term_id="node" >}}.  Nodes that host workloads are referred to as worker nodes. There must be at least one worker node within a cluster. 
```

Comment:
No, this isn't correct. A cluster with zero worker nodes isn't very useful, but it's still technically a cluster (maybe you are auto scaling).

Reply:
Ah - OK.  I kept this as it was part of the previous definition :)